xgi/convert.py

```diff
@@ -302,3 +306,103 @@ def to_incidence_matrix(H, sparse=True, index=False):
     from_incidence_matrix
     """
     return xgi.incidence_matrix(H, sparse=sparse, index=index)
+
+
+def from_bipartite_network(G, create_using=None, dual=False):
+    """
+    Create a Hypergraph from a NetworkX bipartite graph.
+
+    Parameters
+    ----------
+    G : nx.Graph()
+        A networkx bipartite graph. Each node in the graph has a property
+        'bipartite' taking the value of 0 or 1 indicating the type of node.
+
+    create_using : Hypergraph constructor, optional
+        The hypergraph object to add the data to, by default None
+    
+    dual : bool, default : False
+        Whether to get edges from bipartite=0 and nodes from bipartite=1
```

Comment:
"If True, get edges from bipartite=0 and nodes from bipartite=1"
would that prevent any confusion?